core/templates/pages/topic-editor-page/modal-templates/create-new-story-modal.component.spec.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for CreateNewStoryModalController.
+ */
+
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { CreateNewStoryModalComponent } from './create-new-story-modal.component';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { ImageLocalStorageService } from 'services/image-local-storage.service';
+import { StoryEditorStateService } from 'pages/story-editor-page/services/story-editor-state.service';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { FormsModule } from '@angular/forms';
+import { TopicEditorStateService } from '../services/topic-editor-state.service';
+import constants from 'assets/constants';
```

Comment:
Please try to use `AppConstants` instead of `constants`.

Reply:
Done